sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+Both Wark's antipodality and Martin Arnold's rat-drifting draw from the Situationists and their method of the _dérive_, "a technique of rapid passage through varied ambiances... [involving] playful-constructive behavior and awareness of psychogeographical effects" (Debord). I suggest, then, that like cities, disciplines "have psychogeographical contours, with constant currents, fixed points and vortexes that strongly discourage entry into or exit from certain zones" and which must be explored and experienced with both "this letting-go and its necessary contradiction... the dominance of psychogeographical variations by knowledge and calculation" (Debord).
+
+Methodologically, what is key for me to the dérive is its movement, both with and against the gravity of place and action. Rather than spatializing my practice and placing it at a fixed position amongst disciplines in a kind of methodological Venn diagram, I am aiming for a _theory in motion_ that allows assemblages of localized, tactical, context-specific affinities and oppositions to emerge between disciplinary practices. Disciplinary adjacency thus serves as a decentring and a deflection of the temptation to synthesize or subjugate my activities in one discipline to those of another.
+
+#### Practice-Based Research and Adjacency
+
+The subjugation mode is clearly at play also in the increasingly-necessary construction of "practice-based research" in arts scholarship, in which this project ambivalently participates. According to proponents such as Graeme Sullivan, practice-based research asserts that a studio art practice can function as a form of scholarly research, and that "the studio experience is a form of intellectual and imaginative inquiry and is a site where research can be undertaken that is suf­ficiently robust to yield knowledge and understanding that is individually situated and socially and culturally relevant" (90). I do not entirely disagree, though there are fairly profound epistemological and artistic problems to be tangled with. The risk to art in this configuration is that it will always be subjugated to and justified by the regime of scholarly research, reduced, at worst, to a form of "empirical noodling."
```

Comment:
Comment from David Cecchetto by email: "re: 'increasingly-necessary construction of "practice-based research" in arts scholarship' — necessary for whom? when? why? I think you miss a chance to palpate the radicality of your INTD (afforded in part by the privilege of being able to be an artist rather than a labourer, perhaps) to the part of disciplines that are co-constitutive of institutions of a certain type."